package.json

```diff
@@ -21,7 +21,7 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": "^10 || ^12"
+    "node": "^10 || ^12 || ^13"
```

Comment:
how come you added 13? it is not stable..

Reply:
It is indeed not stable, but it is the base ground for node 14 and I think we should be already prepared for node 14 when it comes out in a month or so. When node 14 will be out, we will remove support for node 13.
In addition, if someone does use node 13, why should we not support it?
Thoughts?

Reply:
I haven't read the spec for node 14 so I am not aware for the big changes yet.